events/2020-online-get-together.md

```diff
@@ -51,7 +48,8 @@ Wednesday, 2 December:
 * 12:30 **Discussion sessions**
 * 15:00 Close
 
-#### Proposal submissions ####
+
+### Proposal submissions
 
 There will be a lightweight abstract
 submission system, if you would like to submit events.
```

Comment:
[Abstract submission form](https://indico.neic.no/event/146/overview) is open. Submission deadline: 9th November 2020, 23:59 CET.